### ISSUE TITLE
Warning if import from Jupyter notebook

### ISSUE THREAD

**Issue by tunnell** [rootpy/rootpy#680]
I get the following warning if I import `rootpy` from an IPython notebook:

```/Users/tunnell/anaconda3/envs/pax/lib/python3.4/site-packages/IPython/kernel/__init__.py:13: ShimWarning: The `IPython.kernel` package has been deprecated. You should import from ipykernel or jupyter_client instead.
  "You should import from ipykernel or jupyter_client instead.", ShimWarning)```

Here is the information on my setup:

```
The version of the notebook server is 4.0.6 and is running on:
Python 3.4.3 |Anaconda 2.4.0 (x86_64)| (default, Oct 20 2015, 14:27:51) 
[GCC 4.2.1 (Apple Inc. build 5577)]

Current Kernel Information:

Python 3.4.4 |Anaconda 2.4.0 (x86_64)| (default, Jan  9 2016, 17:30:09) 
Type "copyright", "credits" or "license" for more information.

IPython 4.0.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.
%guiref   -> A brief reference about the graphical user interface.
```

**Comment by ndawe** [comment]
Just noticed this the other day too. Will fix. Thanks for reporting!

**Comment by DCichon** [comment]
This could likely be fixed by replacing line 13 in `rootpy/rootpy/__init__.py`:
```python
from IPython.kernel.zmq.iostream import OutStream
```
with
```python
# try to import OutStream from ipykernel if possible
try:
    from ipykernel.iostream import OutStream
except ImportError:
    from IPython.kernel.zmq.iostream import OutStream
```
This will still give a deprecation warning if _ipykernel_ is not installed (compare with this [_astropy_ pull request](https://github.com/astropy/astropy/pull/4279)), but it should at least minimize the number of occurrences.

**Comment by ndawe** [comment]
Sorry for the delay. Should be fixed in #685. Will merge as soon as tests are green.
